docs/LANGUAGE_TRANSLATION.md

```diff
@@ -92,7 +92,12 @@ If the language translator contributed their translations using a dedicated Goog
 
 Be sure to add a field for "locale", instead of "id".
 
-If the language translator contributed their translation in a Google Sheet with all other translations, you can copy the new translation into a new dedicated spreadsheet and use the above instructions to create the json file. Make sure to copy the ID column, the English column, then the column of the new language translations. This is the only way for the `/bin/google-sheet-with-translation-to-json` to work correctly.
+If the language translator contributed their translation in a Google Sheet with all other translations, you can instead use the `bin/tsv-with-translation-to-json` script. Download the Google Sheet as a TSV (tab-separated-values) file. Copy the file into the `bin` folder. Count the columns (0-indexed) to find the column number of the language you want to use, then call `bin/tsv-with-translation-to-json` in this format:
```

Comment:
I wonder if we could make this whole section (including above) about adding a file a little cleaner? Maybe something like this? Feel free to adjust, I was just thinking of ways to make it clear there are a few different way to do it, cause as I was reading through it initially sounds like it's required to make a copy of `eng.json`. And then also - since right now all the translations are in one file, most people would be able to skip past the "dedicated google sheet" option! What do you think, though?

#### Create a new translation file

##### Manually 
To add a new language, you should first create a copy of [`eng.json`](https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/blob/master/src/i18n/eng.json) and change the filename to the 3-letter [`ISO 639-2`](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) code.

From there you can translate each term on the right side to the corresponding language.

##### Programmatically
If the language translator contributed their translations using 
###### a dedicated Google Sheet for the language:
 you can use `bin/google-sheet-with-translation-to-json` to generate the initial JSON file. For example, for the [Hindi Translation](https://docs.google.com/spreadsheets/d/1gI9Sdbxbl2jy08rjmJn3MTDeuHM4YzeRcihSaWUANbA/edit), we followed [a tutorial](https://www.freecodecamp.org/news/cjn-google-sheets-as-json-endpoint/) to configure the sheet, then the following:

    export SHEET_ID=1gI9Sdbxbl2jy08rjmJn3MTDeuHM4YzeRcihSaWUANbA # The identifier from the URL for Google Sheet
    ./bin/google-sheet-with-translation-to-json > src/i18n/hin.json

Be sure to add a field for "locale", instead of "id".

###### a Google Sheet with all other translations:
you can instead use the `bin/tsv-with-translation-to-json` script. Download the Google Sheet as a TSV (tab-separated-values) file. Copy the file into the `bin` folder. Count the columns (0-indexed) to find the column number of the language you want to use, then call `bin/tsv-with-translation-to-json` in this format:
`bin/tsv-with-translation-to-json [tsv filename] [column number] > [XXX.json]`
So if the TSV file name is `translations.tsv` and you want the Hindi language that's in column 5, you would type:
`bin/tsv-with-translation-to-json ./bin/translations.tsv 5 > src/i18n/hin.json`

If a language doesn't have a translation, the script will not include the id for that word. `translator.js` will use the default english translation if there is not one for the selected language.

Reply:
Yeah good point! It is kinda messy.
Maybe have the headings be something like "If you are doing the translation", "Pulling from a dedicated Google Sheet for the Language" and "Pulling from a Google Sheet with all translations"? Since I wouldn't have known if I was doing it manually or programatically when I first started this?

Reply:
Updated the doc to make it more clear (hopefully). Let me know if you think that change fixes it or if you think we should add something else?

Reply:
Totally! Yeah, super good point!! Awesome thank you :)